internal/provider/incident_workflow_resource.go

```diff
@@ -213,3 +262,107 @@ func (r *IncidentWorkflowResource) buildModel(workflow client.Workflow) *Inciden
 	}
 	return model
 }
+
+func (r *IncidentWorkflowResource) buildConditionGroups(groups []client.ExpressionFilterOptsV2) IncidentEngineConditionGroups {
+	var out IncidentEngineConditionGroups
+
+	for _, g := range groups {
+		conditions := []IncidentEngineCondition{}
+
+		for _, c := range g.Conditions {
+			conditions = append(conditions, IncidentEngineCondition{
```

Comment:
Nit, can we re-order this so subject is first? In my brain (and our UI) a condition is always subject, operation, param binding